Node_Editor/Framework/NodeKnob.cs

```diff
@@ -262,4 +262,24 @@ private static int getRotationStepsAntiCW (NodeSide sideA, NodeSide sideB)
 
 		#endregion
 	}
+
+    public static class NodeKnobUtilities
+    {
+        public static Node GetNodeKnob(this Node node, NodeKnob knob)
+        {
+            if (knob == null)
+                return null;
+            else if (knob is NodeOutput)
+            {
+                if (((NodeOutput)knob).connections.Count > 0)
+                    return ((NodeOutput)knob).connections[0].body;
+            }
+            else if (knob is NodeInput)
+            {
+                if (((NodeInput)knob).connection != null)
+                    return ((NodeInput)knob).connection.body;
+            }
+            return null;
+        }
+    }
 }
```

Comment:
Yup sure, it's just the way I generally do, helps in looking up changes quickly and identifying additional functionality